src/components/CreateResetPasswordLoader/CreateResetPasswordLoaderCtrl.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import {
+  withRouter,
+  Redirect,
+} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { isArray } from 'lodash';
+
+import CreateResetPasswordLoader from './CreateResetPasswordLoader';
+import {
+  defaultErrorCodes,
+} from '../../constants';
+import { stripesShape } from '../../Stripes';
+
+class CreateResetPasswordLoaderCtrl extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        token: PropTypes.string.isRequired,
+      }).isRequired,
+    }).isRequired,
+    stripes: stripesShape.isRequired,
+  };
+
+  state = {
+    isLoaded: false,
+    isValidToken: false,
+    errorCodes: [],
+    actionId: 0,
+  };
+
+  async componentDidMount() {
+    const {
+      match: {
+        params: {
+          token,
+        },
+      },
+      stripes: {
+        okapi: {
+          url,
+          tenant,
+        },
+      },
+    } = this.props;
+
+    try {
+      const response = await fetch(`${url}/bl-users/password-reset/validate`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-okapi-token': token,
+          'x-okapi-tenant': tenant,
+        },
+      });
+
+      await this.handleResponse(response);
+    } catch (error) {
+      const defaultServerErrorCode = defaultErrorCodes.DEFAULT_SERVER_ERROR;
+
+      this.handleDefaultLinkError(defaultServerErrorCode);
+    }
+  }
+
+  handleDefaultLinkError = errorCode => {
```

Comment:
If you use arrow functions approach in the component please do it for all similar methods